src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -177,6 +188,48 @@ def new_forward(*inputs):
 
         layer.module.forward = new_forward
 
+    def export_model(self, model_path, mask_path=None, onnx_path=None, input_shape=None):
+        """
+        Export pruned model weights, masks and onnx model(optional)
+
+        Parameters
+        ----------
+        model_path : str
+            path to save pruned model state_dict
+        mask_path : str
+            (optional) path to save mask dict
+        onnx_path : str
+            (optional) path to save onnx model
+        input_shape : list or tuple
+            input shape to onnx model
+        """
+        assert model_path is not None, 'model_path must be specified'
+        for name, m in self.bound_model.named_modules():
+            mask = self.mask_dict.get(name)
+            if mask is not None:
+                mask_sum = mask.sum().item()
+                mask_num = mask.numel()
+                _logger.info('Layer: %s  Sparsity: %.2f', name, 1 - mask_sum / mask_num)
+                print('Layer: %s  Sparsity: %.2f' % (name, 1 - mask_sum / mask_num))
+                m.weight.data = m.weight.data.mul(mask)
+            else:
+                _logger.info('Layer: %s  Sparsity: %.2f', name, 0)
```

Comment:
it would be better to show this layer is not compressed, for example: 'Not compressed Layer: .....'